tasks.py

```diff
@@ -157,19 +157,18 @@ def pyright(
     context.run('pyright', pty=pty)
 
 
-@task
-def format_check(context):
-    """Run formatting checks."""
-    context.run('ruff check --select I')
```

Comment:
This is unnecessary! Instead, by adding "I" to the toml file, you now get this normally just using `ruff format --check`